### PR TITLE
All: Make more text translatable

### PR DESCRIPTION
**Pull request by wh201906**
Some texts are translatable, but they are not wrapped with `_()`

### CODE REVIEW

**Comment by wh201906** [comment]
I don't know why the run failed
https://github.com/laurent22/joplin/actions/runs/4299993188/jobs/7495769445
The settings panel should have nothing to do with `components/NoteEditor/CodeMirror/markdownCommands`

File: packages/lib/models/Setting.ts [hunk 1]
```diff
@@ -1020,8 +1020,8 @@ class Setting extends BaseModel {
 				section: 'note',
 				public: false,
 				appTypes: [AppType.Mobile],
-				label: () => 'Opt-in to the editor beta',
-				description: () => 'This beta adds list continuation and syntax highlighting. If you find bugs, please report them in the Discourse forum.',
+				label: () => _('Opt-in to the editor beta'),
+				description: () => _('This beta adds list continuation and syntax highlighting. If you find bugs, please report them in the Discourse forum.'),
```
Comment (personalizedrefrigerator) [inline packages/lib/models/Setting.ts]: Thank you for noticing this! I think some of these strings aren't user-facing, however.

`editor.beta` has `public: false`, so shouldn't be visible to the user. Thus, these strings shouldn't need to be translated.

Comment (wh201906) [reply packages/lib/models/Setting.ts]: OK

**Comment by laurent22** [comment]
Thanks for adding this but usually it's on purpose that certain strings aren't translated. We do this for things like beta feature, advanced features or things we might want to remove. This is to reduce the burden on translators so that they don't have to translate things that might be removed later on.

From the look of it, this applies to all the strings that you've added so I think we can close this PR

**Pull request closed without merging**